src/Application/Documents/Queries/GetDocumentsByTitle/GetDocumentsByTitleQuery.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Application.Common.Interfaces;
+using Application.Common.Mappings;
+using Application.Common.Models;
+using Application.Common.Models.Dtos.Physical;
+using AutoMapper;
+using AutoMapper.QueryableExtensions;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace Application.Documents.Queries.GetDocumentsByTitle;
+
+public record GetDocumentsByTitleQuery : IRequest<PaginatedList<DocumentDto>>
+{
+    public Guid UserId { get; init; }
```

Comment:
Imagine: I am a member of HR Department and somehow I know the Guid of a Admin user through a response message. Then I can inject the Guid of that Admin user in `UserId` to see what doesn't available for me!

Reply:
UserId is a user-controlable field. We can fix this by using any kind `user identity value` through JWE or something similar instead of use it directly from a parameter in request message